.gitlab/tests.yml

```diff
@@ -8,6 +8,8 @@ variables:
   needs: []
   parallel: 4
   script:
+    - export PYTEST_ADDOPTS="${PYTEST_ADDOPTS} --ddtrace"
+    - export _DD_CIVISIBILITY_USE_CI_CONTEXT_PROVIDER=true
```

Comment:
Do we set this for hatch but not for riot because it's already set in [riotfile.py](https://github.com/DataDog/dd-trace-py/blob/main/riotfile.py#L101)?

Reply:
Correct.